### ISSUE TITLE
Entries not displaying

### ISSUE THREAD

**Issue by yipiash** [CalderaWP/Caldera-Forms#1631]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 4.7.5PHP Version: 7.0.12MySQL Version: 5.6.32Caldera Forms Version: 1.5.0.10WP_DEBUG:

## Does Your Issue Presist When You Disable All Other Plugins and Switch To The Default Theme?
Yes

## What Is The Unexpected Beahviour?
Entries are not showing in my admin area. I can see the entry number, but when I click on “entries” under a Form, nothing displays.
I’ve read similar topics in that forum but couldn’t solve the problem. 

## What PHP Errors Have You Logged While Reproducing This Bug?
Also I have got my debug log using these codes:

```
define( 'WP_DEBUG', true );
define( 'WP_DEBUG_DISPLAY', false );
define( 'WP_DEBUG_LOG', true );
```

I’ve uploaded my debug.log file here: https://drive.google.com/open?id=0B_LuzeABk2BCVlBVQlF0RjVmRnM

## What JavaScript Errors Have You Seen While Reproducing This Bug?
On my form page (frontend), My console looks like this:
https://drive.google.com/open?id=0B_LuzeABk2BCdmo2TFJiaVlhWHc

**Comment by Shelob9** [comment]
Can you confirm that the "wp_cf_form_entries" (might have a different prefix than wp_" table exists in the database please? phpmyadmin or similar should show you all of the I've seen a few reports that would indicate its not being created in some situations, can't figure out why, but this would help confirm the bug and explain why you can't store entries.

**Comment by yipiash** [comment]
I couldn't find "wp_cf_form_entries" in the database.

**Comment by Shelob9** [comment]
@yipiash Thanks for checking. I'm not sure why this is happening to some users, beacuse it's not happening to me on a new install. But, I'm going to work this week to make sure that we have a fallback in place if they don't get added: https://github.com/CalderaWP/Caldera-Forms/issues/1637#issuecomment-306055661

**Comment by Shelob9** [comment]
Close via fixes in #1637

**Comment by Shelob9** [comment]
See: https://wordpress.org/support/topic/known-issue-entries-not-showing-saving-missing-tables/
